src/components/pages/content/ContentObjectGroups.js

```diff
@@ -55,41 +57,71 @@ class ContentObjectGroups extends React.Component {
   }
 
   render() {
-    let groupCount = Object
-      .values(this.props.groupPermissions)
-      .filter(group => group.permissions.includes(this.state.view))
-      .length;
+    const GroupCount = () => {
+      return Object
+        .values(this.props.objectStore.objectGroupPermissions)
+        .filter(group => group.permissions.includes(this.state.view))
+        .length;
+    };
+
+    let groupsButton;
+    if(this.props.objectStore.object.isOwner || (this.props.objectStore.object.isNormalObject && this.props.objectStore.object.permission !== "owner" && this.props.objectStore.object.canEdit)) {
+      groupsButton = (
+        <Action onClick={() => this.setState({showGroupForm: true})}>
+          New access group permission
```

Comment:
I think we should change this to Manage Group Permissions. Also button text should be capitalized.